modules/openy_features/openy_media/config/install/image.style.full.yml

```diff
@@ -0,0 +1,14 @@
+langcode: en
```

Comment:
Image style should live in `openy_media_image`, because it's being used only there.